File: Interpreter C#/Interpreter/Command.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Interpreter
{
    abstract class Command
    {
        private String key, description;
        public Command(String key, String description) { this.key = key; this.description = description; }
        public abstract void execute();
        public String getKey() { return key; }
        public String getDescription() { return description; }
    }
}

File: MAP/Interpretor2/src/Model/Command.java
package Model;

public abstract class Command {

	private String key;
	private String description;
	
	public Command(String k, String d)
	{
		this.key=k;
		this.description=d;
	}
	
	public abstract void execute();
	public String getKey()
	{
		return this.key;
	}
	public String getDescription()
	{
		return this.description;
	}
}
